azure/azure-release-template.yml

```diff
@@ -70,29 +71,27 @@ extends:
             if [ "$APIGEE_ENVIRONMENT" == "int" ]
             then
               export IDP_URL="https://nhsd-apim-testing-$APIGEE_ENVIRONMENT-ns.herokuapp.com"
+              docker pull artronics/nhsd-login-docker:latest > /dev/null 2>&1
```

Comment:
Would it be better if we didn't redirect stderr to stdout? I think that way we'd still be able to see if something goes wrong

Reply:
Also could move this to line 70 to avoid duplicating it